src/test/java/HW20.java

```diff
@@ -0,0 +1,19 @@
+import com.fasterxml.jackson.databind.ser.Serializers;
+import org.testng.annotations.Test;
+
+public class HW20 extends BaseTest {
```

Comment:
**According to the Java style guide:** 

1. Class names should be nouns, in mixed cases with the first letter of each internal word capitalized. Interfaces names should also be capitalized just like class names.
2. Use whole words and must avoid acronyms and abbreviations.

For example `public class HomeWork20`